iTaylorCodes/Gamr
# Setup Code Coverage

### What does this PR do?
- This Pull Request sets up code coverage using Coveralls.
- This PR closes #5 
### Description of Task to be completed
- Use a service called Coveralls to setup and track code coverage ensuring all code is fully covered by tests. Add Coveralls requirements to requirements.txt. Setup script in travis.yml file to run tests through Coveralls service. Add badge to README.md file to show percentage of code covered. Create .coveragerc file to configure coverage data sent to Coveralls.
### How should this be manually tested?
- Run a build in Travis CI and verify it pushes coverage report to Coveralls service.
